README.md

```diff
@@ -1,4 +1,8 @@
-# Lawnicons
+<p align="center">
+<img src="docs/images/repo_cover.png" alt="Lawnicons repository cover" />
+</p>
+
+<br/>
```

Comment:
```suggestion
```

Looks redundant.

Reply:
I don't know a way to control margins more flexibly, and default ones don't fit well.